### PR TITLE
Use build tags to mark integration tests

### PR DESCRIPTION
**Pull request by eqrx**
### Check List Before Merging

- [ ] This PR passes all pre-commit hook validations.
- [ ] This PR is fully tested and regression tests are included.
- [ ] Relevant documentation has been updated.

### CODE REVIEW

**Comment by erdii** [comment]
ELI5: What do we gain from this?

**Comment by eqrx** [comment]
@erdii, We only have to specify submodules to run tests in, not all the directories in the top level. for a longer time we were not running unit tests in cmd

**Comment by erdii** [comment]
Ok got it.
Doesn't this PR change mage test:unit to not run the unit tests for `./cmd/...`?

**Comment by eqrx** [comment]
@erdii: it runs  all the tests via `./...`

**Review by erdii — APPROVED** [review]
